security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -45,6 +45,9 @@ const (
 	// pass credential token from envoy to SDS.
 	// TODO(quanlin): update value after confirming what headerKey that client side uses.
 	credentialTokenHeaderKey = "access_token"
+
+	// secretName is the name of secret that deliver from SDS service to envoy.
```

Comment:
This "SPKI" name is just the placeholder @mandarjog used for demo. Different envoy should fetch different secrets for their key cert.

Reply:
I know, current agreement with @JimmyCYJ is we hard-coded this name for EAP, probably in the future client side to pass this name to sds.

Reply:
Hard coded name won't work for EAP.

AFAIK, we plan to use secret name to identify different service account. The secret name will be applied based on the deployment YAML annotation and pilot code change.